chia/server/start_service.py

```diff
@@ -51,7 +52,7 @@ def __init__(
         rpc_info: Optional[Tuple[type, int]] = None,
         parse_cli_args=True,
         connect_to_daemon=True,
-        handle_signals=True,
+        running_new_process=True,
```

Comment:
I might go with `manage_process_state` or something perhaps?  I also somewhat wonder if this should just always be handled outside of the service.  But, I'm excited to have this improvement.

Reply:
yeah, I agree that it probably belongs closer to the code spawning new processes, rather than here